docs/02-app/01-building-your-application/08-deploying/02-static-exports.mdx

```diff
@@ -0,0 +1,370 @@
+---
```

Comment:
Is this file supposed to be here? 

I believe it's already live https://nextjs.org/docs/app/building-your-application/deploying/static-exports